velox/expression/CoalesceExpr.h

```diff
@@ -53,6 +53,9 @@ class CoalesceCallToSpecialForm : public FunctionCallToSpecialForm {
       std::vector<ExprPtr>&& compiledChildren,
       bool trackCpuUsage,
       const core::QueryConfig& config) override;
+
+ private:
+  ExprPtr optimize(std::vector<ExprPtr>& compiledChildren);
```

Comment:
Assuming this method doesn't access member variables, it can be moved to a free-standing function in .cpp file. Ditto other similar methods.